code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -266,15 +263,15 @@
 		/obj/item/scalpel,
 		/obj/item/circular_saw,
 		/obj/item/bonesetter,
-		/obj/item/extinguisher/mini,
 		/obj/item/roller/robo,
 		/obj/item/borg/cyborghug/medical,
 		/obj/item/stack/medical/gauze/cyborg,
 		/obj/item/stack/medical/bone_gel/cyborg,
 		/obj/item/organ_storage,
-		/obj/item/borg/lollipop)
+		/obj/item/borg/lollipop,
+		/obj/item/gun/magic/staff/healing)
 	radio_channels = list(RADIO_CHANNEL_MEDICAL)
-	emag_modules = list(/obj/item/reagent_containers/borghypo/hacked)
+	emag_modules = null
```

Comment:
by the way, it'd also be nice if you could set the default cyborg cell type in https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/silicon/robot/robot.dm to be of a self-recharging variety, so that you don't have to add out of place cyborg rechargers to the map

Reply:
**ABOUT THAT, CHIEF**